contents/blog/posthog-vs-ga4.md

```diff
@@ -0,0 +1,536 @@
+---
+date: 2023-10-12
+title: "In-depth: PostHog vs Google Analytics 4"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-ga4/posthog-vs-ga4.jpeg
+featuredImageType: full
+author:
+    - lior-neu-ner
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+Want to understand how PostHog and Google Analytics 4 (GA4) are different? Here's the short answer:
+
+- **Google Analytics** is designed for marketers to understand user traffic, performance of different marketing channels, and return on ad spend.
```

Comment:
Maybe mention e-commerce here? Seems like a big focus for them and not for us.

Reply:
I touch on this below